### DISCUSSION TITLE
Sample API execution failure

### DISCUSSION THREAD

**Discussion by RahulSatanure** [ONDC-Official/ONDC-FIS-Specifications#14] [Q&A]
While executing API's provided on Swagger UI , we are receiving status code : 
405 and Error response as: Method Not Allowed.

Please provide expected response.

**Comment by 92shreyansh** [comment]
Currently the APIs are not hosted, The swagger UI is used to document this APIs only not to validate/test against a hosted sandbox service. The APIs for Consumer and Provider would be implemented by BAP and BPP respectively Gateway is a broadcaseter service only. As ONDC is a Network of these BAPs and BPPs, APIs would not be implemented by ONDC. We are currently working on mock api to have a look and feel for these APIs as static stub responses.
